type/string.md

```diff
@@ -5,7 +5,7 @@
 刚刚学习Rust的同学可能会被Rust的字符串搞混掉，比如`str`，`String`， `OsStr`， `CStr`，`CString`等等……
 事实上，如果你不做FFI的话，常用的字符串类型就只有前两种。我们就来着重研究一下Rust的前两种字符串。
 
-你要明白的是，Rust中的字符串实际上是被编码成UTF-8的一个Unicode字符数组。这么说比较拗口，简单来说，Rust字符串内部存储的是一个UTF-8数组，但是这个数组是Unicode字符经过编码得来的。因此，可以看成Rust原生就支持Unicode字符集（Python2的码农泪流满面）。
+你要明白的是，Rust中的字符串实际上是被编码成UTF-8的一个字节数组。这么说比较拗口，简单来说，Rust字符串内部存储的是一个u8数组，但是这个数组是Unicode字符经过UTF-8编码得来的。因此，可以看成Rust原生就支持Unicode字符集（Python2的码农泪流满面）。
```

Comment:
翻了标准库
`/// Here, there's no need to allocate more memory inside the loop.
#[derive(PartialOrd, Eq, Ord)]
#[stable(feature = "rust1", since = "1.0.0")]
pub struct String {
    vec: Vec<u8>,
}`